src/mem/forward.h

```diff
@@ -73,7 +73,7 @@ void memFreeBuf(size_t size, void *);
 void memFreeRigid(void *, size_t net_size);
 FREE *memFreeBufFunc(size_t size);
 int memInUse(mem_type);
-void memCheckInit(void);
+void memDataInit(mem_type, const char *, size_t, int, bool doZero = true);
```

Comment:
Please remove this static function declaration from forward.h and src/tests/stub_libmem.cc files (at least). AFAICT, those no-longer-wanted lines were resurrected in [b1bc5f7](https://github.com/squid-cache/squid/pull/1590/commits/b1bc5f74878a177a168d89300533574569cee693).